GetStream/stream-video-flutter
# [302] support log file sharing

### 🎯 Goal

Closes #302 

This PR is also pushes android SDK version to 33

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Logs Sharing</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/1286516/228121273-8d9ed14f-9f7c-49e5-a806-ef89f760d141.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>



### 🧪 Testing

- start a call
- wait until you get connected
- double tap on any spot within the participants area
- tap on `Share` Icon
- choose the way to share
- confirm the file is shared


### ☑️Contributor Checklist

#### General

### ☑️Reviewer Checklist
